.github/workflows/ci.yml

```diff
@@ -4,6 +4,10 @@ jobs:
   cloudshell_open:
     runs-on: ubuntu-latest
     steps:
+    - name: Free disk space
+      run: |
+        docker rmi $(docker image ls -aq)
```

Comment:
this can be done by doing `docker system prune -a -f`

Reply:
although it seems like this has resolved the issue :D

Reply:
```suggestion
        docker system prune -a -f
```